tests/rkt_net_test.go

```diff
@@ -381,7 +381,7 @@ func TestNetTemplates(t *testing.T) {
 		Type: "ptp",
 		Ipam: ipamTemplateT{
 			Type:   "host-local",
-			Subnet: "10.1.3.0/24",
+			Subnet: "11.11.3.0/24",
```

Comment:
There will be no such one. 11.* is normal part of public ipv4 range and this is why this addressing should not be used even only in tests.
10.11.x.x should be ok, but also could mess up with other developer setup...